### PR TITLE
FIX: removing duplicate method `_format_results`

### PR DESCRIPTION
**Pull request by razmikmelikbekyan**
*Issue #, if available:*

*Description of changes:*
Basic PEP8 corrections, 
Removing `_format_results` from `Classifier` and `Detector` since it is moved to mother class.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by Innixma** [comment]
Requesting review from @zhanghang1989 since he primarily worked on these files

**Comment by szha** [comment]
Job PR-301-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-301/1/index.html

**Comment by zhanghang1989** [comment]
@aptsunny Could you review the changes?

**Comment by szha** [comment]
Job PR-301-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-301/2/index.html

**Comment by razmikmelikbekyan** [comment]
@Innixma , @zhanghang1989 , @aptsunny should I change something here ?

**Comment by Innixma** [comment]
@aptsunny Please look into this and review to minimize churn in future merge conflicts

File: autogluon/task/object_detection/detector.py [hunk 1]
```diff
@@ -38,7 +37,6 @@ def evaluate(self, dataset, ctx=[mx.cpu()]):
         """
         args = self.args
         net = self.model
-        batch_size = args.batch_size * max(len(ctx), 1)
```
Comment (aptsunny) [inline autogluon/task/object_detection/detector.py]: Sorry, @Why did you remove the batchsize here? @razmikmelikbekyan

Comment (aptsunny) [inline autogluon/task/object_detection/detector.py]: Sorry,  why did you remove the batchsize here? @razmikmelikbekyan

**Comment by aptsunny** [comment]
> @Innixma , @zhanghang1989 , @aptsunny should I change something here ?

Thanks for the fix!

**Comment by szha** [comment]
Job PR-301-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-301/3/index.html

**Comment by zhanghang1989** [comment]
> > @Innixma , @zhanghang1989 , @aptsunny should I change something here ?
> 
> Thanks for the fix!

Does the PR look good to you? If so, could you approve the PR

File: autogluon/task/image_classification/classifier.py [hunk 2]
```diff
@@ -136,15 +137,15 @@ def predict_img(img, ensemble=False):
...
-                items = len(different_dataset[0])
-                for j in range(items):
-                    result.setdefault(j, []).append(different_dataset[i][j])
+                for j in range(len(different_dataset[0])):
+                    result[j].append(different_dataset[i][j])
+
             for c in result.keys():
                 proba_all = sum([*result[c]]) / len(different_dataset)
-                proba_all = mx.nd.array(np.where(proba_all.asnumpy() >= threshold, proba_all.asnumpy(), 0))
+                proba_all = (proba_all > threshold) * proba_all
```
Comment (Innixma) [inline autogluon/task/image_classification/classifier.py]: @aptsunny This is not equivalent functionality to my understanding. Why did you change it?

Comment (aptsunny) [reply autogluon/task/image_classification/classifier.py]: Actually, **proba_all** can all be filtered by threshold, but the modification is faster than before，it should be the equivalent function implement.

Comment (razmikmelikbekyan) [reply autogluon/task/image_classification/classifier.py]: Just a small comment, your replaced `>=` with `>`, is that fine ? The rest behavior will be the same, since all the rest will be `False` which is equivalent to `0`,  so multiplying with proba_all will not change 0s.

Comment (aptsunny) [reply autogluon/task/image_classification/classifier.py]: Thank you very much, I ignored this detail.

**Comment by szha** [comment]
Job PR-301-4 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-301/4/index.html

File: autogluon/task/image_classification/classifier.py (hunk 2 above)
Comment (Innixma) [reply autogluon/task/image_classification/classifier.py]: Thanks for fixing!

**Comment by Innixma** [comment]
Merging, thanks again for your contribution!
